opentasks/src/main/java/org/dmfs/tasks/model/ContentSet.java

```diff
@@ -544,7 +553,7 @@ else if (mBeforeContentValues != null && mBeforeContentValues.get(key) != null)
     }
 
 
-    public void addOnChangeListener(OnContentChangeListener listener, String key, boolean notify)
+    public void addOnChangeListener(@NonNull OnContentChangeListener listener, @Nullable String key, boolean notify)
```

Comment:
I think it doesn't make sense to add a listener for changes to a `null` key. So `key` should be `@NonNull`.

Reply:
There are usages of these methods with `null` key, so that's why I used `Nullable`. I've checked now and `null` actually has a special usage here, see:
https://github.com/dmfs/opentasks/blob/4453861aa3882ae2ed8e6ca48354adfcfdb56563/opentasks/src/main/java/org/dmfs/tasks/model/ContentSet.java#L69-L73

So now I've just added some javadoc to the methods about the key parameter.
Let me know if you'd prefer it to be changed somehow.

Reply:
I right, I forgot about that. Passing null will cause the listener to be called for any update on any key. Did you check if we actually use that?

Reply:
Yes, they are called with `null` key from `ViewTaskFragment` and `EditTaskFragment`.